resources.py

```diff
@@ -2204,96 +369,6947 @@
-\xd7\x51\x25\xbc\xab\x04\x95\x1c\xfe\xe8\x7d\xbc\xa3\x04\x95\x1e\
-\x7e\xba\x04\x2d\xa6\x80\x91\x8b\xfa\x53\xd0\x47\x7e\x97\x0a\x30\
-\x59\xf8\xa3\x43\xeb\x50\x0d\xa6\xae\x00\x67\xdf\xcd\x4b\xb8\x9b\
-\x58\xc2\xdf\x3f\xb8\x99\x2b\x41\x09\x3f\x5b\x82\xa9\xb6\x80\x99\
-\x1b\xb0\x55\xb7\x83\x69\x04\xe8\xd0\x7d\xaf\x38\x21\x94\xf0\xcf\
-\xf9\xec\x59\xb6\x84\xd3\x2b\x40\xd7\xd2\xba\xca\x96\x50\xc2\xcf\
-\x6e\x0e\x4b\xf8\xd9\x12\x1c\xbe\x05\xac\xd8\x48\x75\x96\xf9\x50\
-\x01\x56\xbe\xce\xde\xf5\xd8\xca\x02\x65\x4f\x08\x87\x54\x80\xb4\
-\x27\x74\x3a\x1d\x6f\x59\x8c\xec\xe6\xb0\x84\x9f\x2d\xc1\x2e\x5b\
-\x40\xda\x1f\x55\x74\x3e\x09\x86\x0b\x20\xfc\x5e\x6b\x32\x54\x00\
-\x8f\x63\xf7\x5b\x9f\x12\x7e\x76\x5f\x50\xc2\xcf\x96\xa0\x84\x9f\
-\x2d\xc1\xc3\x3d\x80\x66\x6f\x8d\xca\xf9\x90\x00\xc2\x5f\x67\x42\
-\xf8\xb3\x00\x4a\xfe\x5a\xd5\xa0\x84\x9f\xdd\x17\x94\xf0\xb3\x25\
-\x28\xe1\x67\x4b\x50\xc2\xcf\x96\xa0\x84\x9f\x2d\xc1\xe6\x14\x30\
-\x22\x7c\xc1\xcf\x33\x21\x6c\x65\x51\xb3\x1b\x8c\x7d\xd7\xb2\x84\
-\x9f\x21\xc1\x56\x15\xa9\xd1\x25\x47\xf8\xbd\x2a\x41\x39\xf3\xb3\
-\x25\x28\xe1\x67\x4b\x30\xec\xb5\x71\x8f\x6e\x1b\x6e\x06\x85\x3e\
-\x1b\x88\x39\x20\x00\x01\x40\x00\xfb\x72\x6c\xe3\xa8\x02\xa8\x00\
-\xba\xf3\xe4\xb1\xb1\x8c\x68\xd9\xd7\x0c\xea\xd9\x0f\x40\xdf\xf0\
-\x7f\xdd\x03\x90\x60\xcd\xf0\x2f\x97\x3f\x5c\x09\xfc\xff\x81\xb3\
-\xfd\xcb\x53\x2f\x8d\x7a\xee\xf7\xb9\xed\xf1\x05\x69\xaf\x60\xef\
-\x2c\x9b\x31\xd0\x18\x08\x02\x80\x00\xc8\xe4\xb6\xea\x81\xcd\xfa\
-\xbe\x00\x15\x00\x04\x00\x01\x40\x00\x10\x00\x04\x00\x01\x40\x00\
-\x10\x00\x04\x00\x01\x40\x00\x10\x00\x04\x00\x01\x40\x00\x10\x00\
-\x04\x00\x01\x40\x00\x10\x00\x04\x00\x01\x40\x00\x10\x00\x43\x98\
-\xfe\xed\xe1\xf8\xc2\xe3\xe1\xb0\x05\x80\x00\x98\x5d\x00\xff\x54\
-\xaa\xcf\x9a\xaa\x00\x2a\x40\x2f\x63\x9d\xfd\x2a\x00\xba\x08\xa0\
-\x0a\xcc\xbf\x86\xd5\xfd\x00\x84\xff\x1c\xbb\x5c\x09\xdc\xc2\x15\
-\xc2\xf9\x4e\x9a\x43\x05\x20\xc3\x7c\x95\xf2\x34\x01\x10\xd0\x04\
-\x82\x00\x20\x00\x08\x00\x02\x80\x00\x20\x00\x08\x00\x02\x60\x1e\
-\x3e\x01\xdf\xcb\xf6\xd9\x43\x12\x48\x34\x00\x00\x00\x00\x49\x45\
+\x45\x07\xe3\x04\x1e\x0b\x00\x2c\x27\xcf\xc6\xe2\x00\x00\x02\x82\
+\x49\x44\x41\x54\x78\xda\xed\xdd\x31\x52\xeb\x40\x10\x45\x51\xbb\
+\xcb\x21\xd9\x5f\x22\xeb\x62\x89\x64\xe4\x26\xfa\x45\x11\xa8\x00\
+\x7b\x24\x4d\xcf\x3b\x37\x24\xb0\xd1\xbc\xab\x9e\x6e\x09\xa1\xeb\
+\xfd\x7e\xbf\x20\x97\xb2\x04\x04\x00\x01\x40\x00\x10\x00\x04\x00\
+\x01\x40\x00\xc4\x70\x4b\x3d\xf0\x8f\xb7\x7f\xdf\xae\x80\xbd\xbc\
+\xbe\x5f\x55\x80\xd0\xf0\xb7\x7e\x46\x80\x90\xf0\x93\x25\x28\xe1\
+\x67\x4b\x50\xc2\xcf\x96\xa0\x84\x9f\x2d\xc1\x75\xe5\xdb\xc1\xa3\
+\x42\x5c\x79\x42\x28\xe1\x67\x57\x83\x12\x7e\xb6\x04\x25\xfc\x6c\
+\x09\x4a\xf8\xd9\x12\x94\xf0\xb3\x25\x68\x3f\x05\x9c\x1d\x46\xf7\
+\x09\xa1\x84\x9f\x5d\x0d\x4a\xf8\xd9\x12\x54\x72\xf8\x2f\xaf\xef\
+\xd7\x51\x25\xbc\xab\x04\x95\x1c\xfe\xe8\x7d\xbc\xa3\x04\x95\x1e\
+\x7e\xba\x04\x2d\xa6\x80\x91\x8b\xfa\x53\xd0\x47\x7e\x97\x0a\x30\
```

Comment:
Weird, I have a completely different output for this. If you don't mind, i'll use mine.